incubator/hnc/go.mod

```diff
@@ -14,4 +14,5 @@ require (
 	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
 	sigs.k8s.io/controller-runtime v0.2.2
+	sigs.k8s.io/controller-tools v0.2.0-beta.4 // indirect
```

Comment:
Can you remove this? Read about Go modules to see why this was added, but basically you must have called `go get controller-tools` while in this directory.

Reply:
Thanks! I was unsure as to exactly what this was. I'll do that reading in a bit. Go mods are something I haven't used just yet in my Go career.